### ISSUE TITLE
Base ga4gh PURL does not resolve

### ISSUE THREAD

**Issue by cmungall** [GA4GH-Pedigree-Standard/family_history_terminology#36]
This resolves
http://purl.org/ga4gh/kin.owl

But this does not:
http://purl.org/ga4gh/

Although this is not strictly a KIN problem, AFAICT KIN is the main user of this purl, and many semantic web users would expect to find something in the parent path, and I am not sure where else to report this

FWIW, there is also a PURL registered at https://w3id.org/ga4gh/

https://github.com/perma-id/w3id.org/blob/master/ga4gh

This resolves to a not very useful directory with a single minimal README

**Comment by cmungall** [comment]
@lindsmith @buske any update on this? The monarch team has a lot of experience with w3id just reach out if you need help...

**Comment by buske** [comment]
@cmungall These purl paths were set up by @ametke. Perhaps he has input on this? Should the base path resolve to something ga4gh generic, or something ga4gh pedigree specific, since it looks like those are [the only purls under the ga4gh path](https://purl.archive.org/domain/ga4gh)?

**Comment by ametke** [comment]
Yes, I created those with my account. I'll delete them so someone in GA4GH can take over and redefine them appropriately.

**Comment by ametke** [comment]
There doesn't seem to be an obvious way to transfer ownership of this @buske and I don't think I can (or should) delete them since they are meant to be persistent identifiers.

**Comment by cmungall** [comment]
It seems like there are not in heavy use yet

For the base resolver, I favor w3id.org over purl.org. I use w3id for many projects, it's a really simple and transparent system, all managed by github and PR.

Years ago OBO used purl.org, and it was disastrous, as it became impossible to edit the purls we owned, the system may have improved

whatever the base resolver, the next question is whether to have an overall ga4gh namespace with sub-namespaces underneath, or to have looser coupling e.g. w3id.org/kin

**Comment by cmungall** [comment]
Any objections to using w3id.org?

**Comment by cmungall** [comment]
Just following up on this - any objections to using w3id.org?
